db/models.py

```diff
@@ -14,3 +14,43 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    actors = models.ManyToManyField(Actor, related_name="actor")
+    genres = models.ManyToManyField(Genre, related_name="genre")
```

Comment:
The `related_name` 'genre' for the `genres` field might cause confusion if there are multiple genres related to a movie. Consider using a plural form like 'genres'.